freqtrade/plot/plotting.py

```diff
@@ -64,14 +64,14 @@ def add_indicators(fig, row, indicators: List[str], data: pd.DataFrame) -> make_
     """
     for indicator in indicators:
         if indicator in data:
-            # TODO: Figure out why scattergl causes problems
-            scattergl = go.Scatter(
+            # TODO: Figure out why scattergl causes problems plotly/plotly.js#2284
```

Comment:
I would move the plotly reference to the bollinger band section, as it's very specific to how that is rendered (the bugreport does not appy to the following lines since they don't use `fill="tonexty"`

Reply:
See comment below. Testing confirms that the problem is with `Scattergl`.